### ISSUE TITLE
Home screen widgets for ease of use

### ISSUE THREAD

**Issue by Miroaja** [davidhealey/waistline#821]
The android widget system would be excellent in smoothing out the rough edges of waistlines UX. Having e.g. daily stats right on the home screen, or being able to quickly log a meal from there would help reduce a lot of the friction with meal tracking, especially with waistlines ..unique UI.

**Comment by skisbit** [comment]
Agreed, currently no FOSS app have a widget for calorie-tracking like most proprietary applications do.
It's harder to forget to log something if you're presented with a number when you unlock your screen.

**Comment by Miroaja** [comment]
There are other UI/UX issues with waistline that moved me towards OpenNutriTracker, but tbh waistline has way more functionality which almost make switching away from it impossible

**Comment by sfmiller940** [comment]
This app is developed with Cordova and it seems like [widgets aren't easily possible with Cordova](https://stackoverflow.com/questions/25938933/android-widget-using-cordova).
